_benchmark/installing-benchmark.md

```diff
@@ -152,4 +152,5 @@ See [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/co
 
 ## Next steps
 
-- [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/)
\ No newline at end of file
+- [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/)
```

Comment:
This link doesn't seem to work when I click on the hyperlink

Reply:
Gotcha, I tested out 3 links in the document when I opened them here https://github.com/opensearch-project/documentation-website/blob/benchmark-workflows/_benchmark/creating-a-custom-workload.md but only the workloads repo pulls up. I thought it might work whenever it's deployed to the site.